mon-pix/tests/integration/components/signin-form-test.js

```diff
@@ -53,20 +63,160 @@ describe('Integration | Component | signin form', function() {
       await render(hbs`<SigninForm />`);
 
       // then
-      expect(document.querySelector('div.sign-form__notification-message')).to.not.exist;
+      expect(document.querySelector('div.sign-form__error-message')).to.not.exist;
     });
 
-    it('should display an error if authentication failed', async function() {
-      // given
-      this.set('authenticateUser', sinon.stub().rejects());
-      await render(hbs`<SigninForm @authenticateUser={{this.authenticateUser}} />`);
+    context('When error api occurs', function() {
+
+      it('should display related error message if bad request error', async function() {
+        // given
+        const expectedErrorMessage = ApiErrorMessages.BAD_REQUEST.MESSAGE;
+        const apiReturn = {
+          errors: [{
+            code: 400,
+            detail: expectedErrorMessage,
+            title: 'Bad request error'
+          }]
+        };
+        this.set('authenticateUser', sinon.stub().rejects(apiReturn));
+        await render(hbs`<SigninForm @authenticateUser={{this.authenticateUser}} />`);
+
+        // when
+        await fillIn('input#login', 'wrongemailformat');
+        await fillIn('input#password', 'wrongpasswordformat');
+        await click('button.button');
 
-      // when
-      await click('button.button');
+        // then
+        expect(find('.sign-form__error-message').textContent.trim()).to.equal(this.intl.t(expectedErrorMessage));
+      });
+
+      it('should display related error message if unauthorized error', async function() {
+        // given
+        const expectedErrorMessage = ApiErrorMessages.UNAUTHORIZED.MESSAGE;
+        const apiReturn = {
+          errors: [{
+            status: 401,
+            detail: expectedErrorMessage,
+            title: 'Unauthorized error'
+          }]
+        };
+        this.set('authenticateUser', sinon.stub().rejects(apiReturn));
+        await render(hbs`<SigninForm @authenticateUser={{this.authenticateUser}} />`);
+
+        // when
+        await fillIn('input#login', 'usernotexist@example.net');
+        await fillIn('input#password', 'password');
+        await click('button.button');
+
+        // then
+        expect(find('.sign-form__error-message').textContent.trim()).to.equal(this.intl.t(expectedErrorMessage));
+      });
+
+      it('should display an error if api is not up ', async function() {
```

Comment:
S'il passe sous un tunnel..
```suggestion
      it('should display an error if api cannot be reached up ', async function() {
```

Reply:
Je modifie en reached without up :)